<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window2</class>
 <widget class="QDialog" name="Window2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="ItemPriceL">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item Price :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ChickOpt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chicken Burger üçî</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chicken Wings üçó</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chicken Nuggets üê•</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>whole chicken üêî</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sheesh Tawook üç¢</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="AccBalanceL">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Account Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="ChickOptL">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>100</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string>Chicken Options</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewCart">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>View Cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="testing123">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>101</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="MeatOpt">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Meat Burger üçî</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sausages üå≠</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Meatballs üßÜ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Steak ü•©</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>beef bacon ü•ì</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kabab ü•ô</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="MeatOptL">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Meat Options</string>
   </property>
  </widget>
  <widget class="QLabel" name="NumAddItemsL">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Number of Added Items :</string>
   </property>
  </widget>
  <widget class="QLabel" name="ItemPrice">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AddTocart">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add To Cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccBalance">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="NumAddItems">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
